cityofaustin/atd-vz-data
# Remove the "Inconsistent KSI Counts" and "Changes" features

## Associated issues
Closes https://github.com/cityofaustin/atd-data-tech/issues/11959 and https://github.com/cityofaustin/atd-data-tech/issues/18327

## Testing
**URL to test:** 
Local

**Steps to test:**
1. Check out this branch and start your data model DB
2. See that the Changes and Inconsistent KSI Counts features are gone now!!


---
#### Ship list
